content/docs/concepts/security/index.md

```diff
@@ -10,98 +10,160 @@ aliases:
     - /docs/concepts/security/rbac/
 ---
 
-Istio aims to enhance the security of microservices and their communication without requiring service code changes. It is responsible for:
+The benefits of taking a monolithic application and breaking it down into atomic services are striking:
+microservices have better agility, better scalability, and better ability to reuse services.
+However, microservices also have particular security needs.
+For example, to defend against the man-in-the-middle attack, they need traffic encryption;
+to provide flexible service access control, they need mutual TLS and fine-grained access policies;
+to audit who did what at what time, they need auditing tools.
+Istio Security’s aim is to provide a comprehensive security solution to solve all these issues.
 
-* Providing each service with a strong identity that represents its role to enable interoperability across clusters and clouds
+This page gives an overview of how Istio security can be leveraged to secure your services, wherever you run them.
+In particular, Istio security mitigates both insider and external threats against your data, endpoints, communication and platform.
 
-* Securing service to service communication and end-user to service communication
+{{< image width="80%" ratio="56.25%"
+    link="./overview.svg"
+    alt="Components making up the Istio security model."
+    caption="Istio Security Architecture"
+    >}}
+
+Istio security provides strong identity, powerful policy, and transparent TLS encryption to protect your services and data
+with AAA (authentication/authorization/audit). Istio security’s aims are:
+
+* **Secure by default**: no changes needed to application code and infrastructure
+
+* **Defence in depth**: integrates with existing security systems to provide multiple layers of defense
```

Comment:
Defence/Defense - I know its a little annoying that we use US English in the website.. :)

Reply:
Done. Didn't notice it :)